migrations/2_deploy.js

```diff
@@ -229,9 +231,10 @@ module.exports = async function(deployer, network, accounts) {
     deployerApp.writeJson();
     console.log(`${'='.repeat(25)} Deployment process finished. ${'='.repeat(25)}`);
   } catch (error) {
+    console.log('\x1b[33m\x1b[41m\x1b[5m', `Error deploying contract`, '\x1b[0m');
     console.log(error);
     deployerApp.print();
     deployerApp.writeJson();
-    console.log(`${'='.repeat(25)} Deployment process FAILED. ${'='.repeat(25)}`);
+    console.log('\x1b[33m\x1b[41m\x1b[5m%s\x1b[0m', `${'='.repeat(25)} Deployment process FAILED. ${'='.repeat(25)}`);
```

Comment:
whats this?

Reply:
It's a color on terminal to identify deployment errors more easily in case of a failure

Reply:
Could we add a custom logger for this? I like that idea

Reply:
Yes I'll add a new story for that task